### PR TITLE
FLW-711 Add support in History for Staking rewards & Slashes type

### PR DESCRIPTION
**Pull request by ERussel**
- update ui for transaction history list

### CODE REVIEW

File: fearless/Modules/Wallet/History/View/HistoryItemTableViewCell.swift [hunk 1]
```diff
@@ -1,33 +1,162 @@
...
-        selectedBackgroundView.backgroundColor = R.color.colorAccent()!.withAlphaComponent(0.3)
-        self.selectedBackgroundView = selectedBackgroundView
+        setupLayout()
+    }
+
+    private func setupLayout() {
+        contentView.addSubview(transactionTypeView)
+
+        transactionTypeView.snp.makeConstraints { make in
+            make.leading.equalTo(contentView).offset(UIConstants.horizontalInset)
```
Comment (ef1rspb) [inline fearless/Modules/Wallet/History/View/HistoryItemTableViewCell.swift:71]: just equalToSuperview()

File: fearless/Modules/Wallet/History/View/HistoryItemTableViewCell.swift [hunk 2]
```diff
@@ -1,33 +1,162 @@
...
+            make.leading.equalTo(transactionTypeView.snp.trailing).offset(UIConstants.horizontalInset / 2.0)
+            make.top.equalTo(contentView).offset(Constants.verticalInset)
+        }
+
+        contentView.addSubview(subtitleLabel)
+
+        subtitleLabel.snp.makeConstraints { make in
+            make.leading.equalTo(transactionTypeView.snp.trailing).offset(UIConstants.horizontalInset / 2.0)
+            make.top.equalTo(titleLabel.snp.bottom)
+            make.bottom.equalTo(contentView).offset(-Constants.verticalInset)
```
Comment (ef1rspb) [inline fearless/Modules/Wallet/History/View/HistoryItemTableViewCell.swift:88]: inset(Constants.verticalInset)
`offset(..)` is used with views in the same view hierarchy, i.e.  `make.leading.equalTo(transactionTypeView.snp.trailing).offset(..)`

File: fearless/Modules/Wallet/History/View/HistoryItemTableViewCell.swift (hunk 1 above)
Comment (ERussel) [reply fearless/Modules/Wallet/History/View/HistoryItemTableViewCell.swift]: applied

File: fearless/Modules/Wallet/History/View/HistoryItemTableViewCell.swift (hunk 2 above)
Comment (ERussel) [reply fearless/Modules/Wallet/History/View/HistoryItemTableViewCell.swift]: fixed

**Review by ef1rspb — APPROVED** [review]

**Pull request merged**